test/metabase/api/table_test.clj

```diff
@@ -664,6 +664,13 @@
                      ["field" nil {:binning {:strategy "num-bins", :num-bins 10}}]}
                    (extract-dimension-options response "price"))))))
 
+      (testing "Relationship fields should not have binning options"
+        (let [fingerprint (db/select-one-field :fingerprint Field {:id (mt/id :venues :price)})]
```

Comment:
This query doesn't do what you think it's doing. You need to write either

```clj
(db/select-one-field :fingerprint Field :id (mt/id :venues :price))
```

or

```clj
(db/select-one-field :fingerprint Field {:where [:=:id (mt/id :venues :price)]})
```

Maps passed to Toucan function are just treated as raw HoneySQL. `:id` in a HoneySQL map just gets ignored

Reply:
Can you add an assertion like  

```clj
(is (seq (extra-dimension-options response "id")))
```

_before_ adding the fingerprint information? So the whole test looks like

1. before changing anything
   1. test that dimension options get returned
1. add Fingerprint info
   1. test that dimension options no longer get returned

otherwise for all we know the change you made adding `:fingerprint` info had nothing to do with anything and the dimension options wouldn't have been returned either way